build.gradle

```diff
@@ -118,10 +123,14 @@ configure(scalaProjects) {
         scalaCompileOptions.deprecation = false
     }
 
+
+    tasks.withType(GenerateModuleMetadata) {
```

Comment:
Added to resolve the exception `Signing Gradle Module Metadata is not supported for snapshot dependencies.`